### PR TITLE
Bahasa Melayu translations

### PR DESCRIPTION
**Pull request by apaoww**
Translate yii2 message into Malaysian language which is Bahasa Melayu

### CODE REVIEW

File: framework/messages/config.php [hunk 1]
```diff
@@ -7,7 +7,7 @@
     'messagePath' => __DIR__,
     // array, required, list of language codes that the extracted messages
     // should be translated to. For example, ['zh-CN', 'de'].
-    'languages' => ['ar', 'az', 'bg', 'ca', 'cs', 'da', 'de', 'el', 'es', 'et', 'fa-IR', 'fi', 'fr', 'he', 'hu', 'id', 'it', 'ja', 'kk', 'ko', 'lt', 'lv', 'nl', 'pl', 'pt', 'pt-BR', 'ro', 'ru', 'sk', 'sl', 'sr', 'sr-Latn', 'sv', 'th', 'tj', 'uk', 'vi', 'zh-CN','zh-TW'],
+    'languages' => ['ar', 'az', 'bg', 'ca', 'cs', 'da', 'de', 'el', 'es', 'et', 'fa-IR', 'fi', 'fr', 'he', 'hu', 'id', 'it', 'ja', 'kk', 'ko', 'lt', 'lv', 'ms-MY', 'nl', 'pl', 'pt', 'pt-BR', 'ro', 'ru', 'sk', 'sl', 'sr', 'sr-Latn', 'sv', 'th', 'tj', 'uk', 'vi', 'zh-CN','zh-TW'],
```
Comment (cebe) [inline framework/messages/config.php]: is the translation specific to `MY` or may it apply in general?
If it is not specific we should prefer `ms` over `ms-MY`.

Comment (apaoww) [inline framework/messages/config.php]: There are other countries using it like Singpore and Brunei.

Comment (samdark) [inline framework/messages/config.php]: Then it should be `ms` so translation could be shared betweed countries. @apaoww could you make additional commit to the same branch changing it? Thanks!

Comment (apaoww) [inline framework/messages/config.php]: Alright then, I will change it from ms-MY to just ms.

**Comment by qiangxue** [comment]
Thanks!

**Comment by enda34** [comment]
tankz[?]


2015-02-07 9:46 GMT+07:00 Qiang Xue <notifications@github.com>:

**Pull request merged**